contrib/install/install.ps1

```
@@ -0,0 +1,168 @@
+    return Join-Path -Path $rootDirectory -ChildPath "downloads"
+}
+
+function Resolve-Platform {
+    if (-not [string]::IsNullOrWhiteSpace($env:OO_INSTALL_PLATFORM)) {
+        return $env:OO_INSTALL_PLATFORM
+    }
+
+    Assert-Windows
+
+    $architecture = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture
+        .ToString()
```

PSScriptAnalyzer (1.25.0)

[error] An expression was expected after '('.

(ExpectedExpression)